Ginger/GingerTest/GingerAutomator.cs

```diff
@@ -96,8 +104,10 @@ private void StartGinger()
                 
 
                 app.HideConsoleWindow();
+                Console.WriteLine("StartGingerUI");
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Remove this logging statement.](https://app.codacy.com/manual/Ginger-Automation/Ginger/pullRequest?prid=4707297)